it/uk/gov/hmrc/tai/integration/TaxCodeMismatchSpec.scala

```diff
@@ -49,6 +53,21 @@ class TaxCodeMismatchSpec extends IntegrationSpec {
       List(500, 501, 502, 503, 504).foreach { status =>
 
         s"return $status when we receive $status downstream" in {
+          val nino = new Generator(new Random).nextNino
+          val npsIabdsUrl = s"/nps-hod-service/services/nps/person/$nino/iabds/$year"
+
+          val npsTaxAccountUrl = s"/nps-hod-service/services/nps/person/$nino/tax-account/$year"
```

Comment:
This block of code is repeated a few times in here - might be nicer to put into a private method